backend/app/tests/conftest.py

```
@@ -1,19 +1,26 @@
 from collections.abc import Generator
 
 import pytest
+import os
```

Ruff (0.12.2)

`os` imported but unused

Remove unused import: `os`

(F401)